django_netjsonconfig/static/django-netjsonconfig/js/widget.js

```diff
@@ -67,13 +67,44 @@
         return editor;
     };
 
+    var evaluateVars = function(data, context){
+        var pattern = new RegExp('\{\{.*?\}\}');
+                if (typeof data === 'object'){
+                    for (let [key, value] of Object.entries(data)){
```

Comment:
do not use `let` or other ES6 constructs, this code cannot assume ES6 is supported.